catalog/transformer/catalog-transformer-streaming-dependencies/pom.xml

```diff
@@ -0,0 +1,265 @@
+                <version>${org.geotools.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.vividsolutions</groupId>
+                <artifactId>jts</artifactId>
+                <version>1.13</version>
+            </dependency>
+            <dependency>
+                <groupId>commons-collections</groupId>
+                <artifactId>commons-collections</artifactId>
+                <version>3.2.2</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-xml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.1.0</version>
+
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <artifactSet>
+                                <includes>
+                                    <include>org.geotools:gt-xml</include>
+                                    <include>org.geotools:gt-api</include>
+                                    <include>org.geotools:gt-main</include>
+                                    <include>jdom:jdom</include>
+                                    <include>org.geotools.xsd:gt-xsd-wfs</include>
+                                    <include>org.geotools.xsd:gt-xsd-filter</include>
+                                    <include>org.geotools:gt-coverage</include>
+                                    <include>javax.media:jai_imageio</include>
+                                    <include>it.geosolutions.imageio-ext:imageio-ext-tiff</include>
+                                    <include>it.geosolutions.imageio-ext:imageio-ext-utilities</include>
+                                    <include>javax.media:jai_codec</include>
+                                    <include>org.jaitools:jt-zonalstats</include>
+                                    <include>org.jaitools:jt-utils</include>
+                                    <include>org.geotools.xsd:gt-xsd-core</include>
+                                    <include>org.geotools:gt-graph</include>
+                                    <include>picocontainer:picocontainer</include>
+                                    <!--<include>xml-apis:xml-apis</include>-->
```

Comment:
Should probably remove these commented out lines.

Reply:
Done